docs/services/frame-system/_index.md

```diff
@@ -91,18 +93,104 @@ You can use [the right-hand rule](https://en.wikipedia.org/wiki/Right-hand_rule)
+Navigate to the **config** tab on your robot's page, select **mode** as **Builder**, and click on **Frame System**.
+
+A 3D visualization of the spatial configuration of the reference frames of components configured on your robot is shown:
+
+![Default frame system configuration grid visualization for a single component, shown in the Frame System Editor](img/frame_system_basic.png)
+
+This tab provides a simple interface for simultaneously viewing and editing the position, orientation, and geometries of a robot's components in the Frame System.
+
+For example:
+
+- You have configured a robot with a [`jetson` board](/components/board), wired to a [`webcam` camera](/components/camera/webcam) and a [`wheeled` base](/components/base/wheeled) with two [motors](/components/motor) driving its wheels.
+
+- You go to the **config** tab of your robot and click on the **Frame System** sub-tab.
+You have not yet added any reference frame configuration to these components, so they are all shown to be located on the default `world` origin point as follows:
+
+  ![Example robot's default frame configuration shown in the Frame System Editor. All components are stuck on top of each other](img/demo_base_unedited.png)
+
+- Then, you measure the distance on the floor from your base to your board and camera setup and find it to be 200 millimeters.
+You add this value to `"X"` in your base's reference frame `Translation` attribute and the Frame System readjusts to show the base's translation:
+
+  ![Base translated 200mm forwards shown in the Frame System Editor](img/demo_base_edited.png)
+
+- You measure the distance from your board to the camera mounted overhead and find it to be 50 millimeters.
+You add this value to `"Z"` in your camera's reference frame `Translation` attribute and the Frame System readjusts to show the camera's translation:
+
+  ![Camera translated 50 mm overhead shown in the Frame System Editor](img/demo_camera_edited_1.png)
+
+- However, the camera doesn't yet display as oriented towards the base.
+You adjust the [orientation vector](/internals/orientation-vector) 0.5 degrees in `"OX"` in your camera's reference frame `Orientation` attribute and the Frame System readjusts to show the camera's orientation:
+
+  ![Camera oriented .5 degrees OX shown in the Frame System Editor](img/demo_camera_edited_2.png)
+
+Now that the Frame System is configured so that the spatial visualization for your robot matches its spatial orientation in real life, you can utilize [Motion Service](/services/motion) methods that take in reference frame information.
+
+### Display Options
+
+Click and drag on the **Frame System** visualization to view the display from different angles, and pinch to zoom in and out:
+
+{{<gif webm_src="img/frame_system_demo.webm" mp4_src="img/frame_system_demo.mp4" alt="The frame system visualization zooming in and out around the example robot with a camera, board, and wheeled base.">}}
```

Comment:
this is very cool - for the future, open up the developer tools and remove the chat bubble when taking these screen recordings (let me know if you want me to show you how). Not worth redoing now, just keep it in mind

Reply:
Oh I didn't think of that thank you!